pages/features.html

```diff
@@ -489,19 +489,19 @@ <h4>Seamlessly integrate with your asset pipeline</h4>
 			<p>
 				Bring your 3D models into your game world with a robust
 				importing pipeline. Take entire scenes — with animated models,
-				lighting, cameras, and physics objects, — and customize how
+				lighting, cameras, and physics objects — and customize how
 				the engine views them. Modify your assets and see changes
 				in the engine immediately.
 				<br><br>
-				<strong>New in 4.0:</strong> Import Blender files directly
-				for fast iterations or keep using familiar glTF and FBX formats.
+				Import Blender files directly
+				for fast iterations or use familiar glTF and FBX formats.
 			</p>
 		</div>
 
 		<div class="card feature-card">
 			<h4>Create animated videos and prerendered cutscenes in engine</h4>
 			<p>
-				<strong>New in 4.0:</strong> With the new movie maker mode you
+				With the movie-maker mode you
 				can record gameplay and scripted scenes from your project at
 				a stable framerate and guaranteed simulation speed. Together
 				with Godot's animation capabilities, make the most out of the
```

Comment:
We tend to capitalize it elsewhere (and write it without a hyphen):

```suggestion
				With the Movie Maker mode, you
				can record gameplay and scripted scenes from your project at
				a stable framerate and guaranteed simulation speed. Together
				with Godot's animation capabilities, make the most out of the
```